guide/how-it-works/liquidity.md

```diff
@@ -0,0 +1,314 @@
+
+<div class="nodes -full-width" markdown="1">
+   <table>
+      <thead>
+         <tr>
+            <th>&nbsp;</th>
+            <th>
+               {% include picture.html
+                  image = "/assets/images/guide/how-it-works/liquidity/node-rachel.png"
+                  retina = "/assets/images/guide/how-it-works/liquidity/node-rachel@2x.png"
+                  alt-text = "Depiction of Rachel and a dedicated hardware lightning node"
+                  width = 236
+                  height = 236
+               %}
+               Routing Node Operator
+            </th>
+            <th>
+               {% include picture.html
+                  image = "/assets/images/guide/how-it-works/liquidity/node-miguel.png"
+                  retina = "/assets/images/guide/how-it-works/liquidity/node-miguel@2x.png"
+                  alt-text = "Depiction of Miguel and a cloud lightning node"
+                  width = 236
+                  height = 236
+               %}
+               Merchant
+            </th>
+            <th>
+               {% include picture.html
+                  image = "/assets/images/guide/how-it-works/liquidity/node-wagner.png"
+                  retina = "/assets/images/guide/how-it-works/liquidity/node-wagner@2x.png"
+                  alt-text = "Depiction of Wagner and a mobile phone lightning node"
+                  width = 236
+                  height = 236
+               %}
+               Wallet User
+            </th>
+         </tr>
+      </thead>
+      <tbody>
+         <tr>
+            <td>What is their incentive?</td>
+            <td>Rachel operates a lightning node so she can earn routing fees for providing liquidity to the network</td>
+            <td>Miguel operates a lightning node so he can receive payments at his business</td>
+            <td>Wagner has a mobile bitcoin wallet that supports lightning</td>
+         </tr>
+         <tr>
+            <td>What's their role in the lightning network?</td>
+            <td>Rachel acts more like a hub for routing other's lightning payments, and is rarely the origin or endpoint</td>
+            <td>Miguel is usually the endpoint for lightning payments</td>
+            <td>Wagner is sometimes the origin and sometimes the endpoint for lightning payments</td>
+         </tr>
+         <tr>
+            <td>Where does it run?</td>
+            <td>Runs on dedicated hardware, like a Raspberry Pi or rack-mounted server</td>
+            <td>Runs on a cloud service provider</td>
+            <td>Runs on Wagner's phone</td>
+         </tr>
+         <tr>
+            <td>When is it online?</td>
+            <td>Always online</td>
+            <td>Always online</td>
+            <td>Only online when Wagner opens the app</td>
+         </tr>
+         <tr>
+            <td>How many lightning channels?</td>
+            <td>Many lightning channels</td>
```

Comment:
```suggestion
            <td>Many</td>
```
Remove the term `lightning channels` from all rows (except for the question) for brevity?

Reply:
Done